src/syscheckd/src/create_db.c

```diff
@@ -1510,10 +1509,9 @@ void fim_print_info(struct timespec start, struct timespec end, clock_t cputime_
             (double)(clock() - cputime_start) / CLOCKS_PER_SEC);
 
 #ifdef WIN32
-    /*
-    mdebug1(FIM_ENTRIES_INFO, fim_db_get_count_file_entry(syscheck.database));
-    mdebug1(FIM_REGISTRY_ENTRIES_INFO, fim_db_get_count_registry_key(syscheck.database) + fim_db_get_count_registry_data(syscheck.database));
-    */
+    mdebug1(FIM_ENTRIES_INFO, fim_db_get_count_file_entry());
+    mdebug1(FIM_REGISTRY_ENTRIES_INFO, fim_db_get_count_registry_key());
```

Comment:
Is it needed to count the entries of the keys table? We are not using this for anything else.

Reply:
It is a debug message which shows both registry values and registry keys count, I think it may be useful information.